src/deserialize.rs

```diff
@@ -41,7 +41,8 @@ pub fn email_address<'d, D>(deserializer: D) -> Result<String, D::Error>
 where
     D: Deserializer<'d>,
 {
-    deserialize(deserializer, validate::email_address, "email address")
+    let email = deserialize(deserializer, validate::email_address, "email address")?;
+    Ok(email.to_lowercase())
```

Comment:
>  seemed cleaner to just make the email lowercase on deserialize

Is it possible to add a test for `UPPERCASE@emails.com` ?

Reply:
Done.

Reply:
Done.

Reply:
Just noting for future reference that although making it lower case at this point is fine for now, longer term we may want to think about keeping it internal to the `Bounces` module.

These email addresses get reported back to the caller via the outgoing notification queue and it might be a less surprising API if the email addresses that go out can be directly string-compared to the ones that were passed in. For the auth server this probably isn't an issue because we'll call it with `normalizedEmail` anyway. But if/when we're open to other callers it might be something to think about.

Reply:
>  But if/when we're open to other callers it might be something to think about.

👍